stdpopsim/catalog/ApiMel/species.py

```diff
@@ -0,0 +1,117 @@
+import stdpopsim
+
+from . import genome_data
+
+# [The following are notes for implementers and should be deleted
```

Comment:
We can delete this comment (and others like it), it's just there as an initial guide for people.